inspec.gemspec

```diff
@@ -38,4 +38,5 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'sslshake', '~> 1'
   spec.add_dependency 'parallel', '~> 1.9'
   spec.add_dependency 'rspec_junit_formatter', '~> 0.2.3'
+  spec.add_dependency 'requests', '~> 1.0.0'
```

Comment:
Do we really need the requests module? It is not maintained for a couple of years anymore. See https://github.com/cyx/requests.